server/index.js

```diff
@@ -43,6 +44,19 @@ app.get('/api/yelp/:businessId', (req, res) => {
     .then(reviews => res.status(200).json(reviews))
     .catch(error => console.error(error));
 });
+
+app.post('/api/twilio/:phoneNumber/:address', (req, res) => {
+  const { phoneNumber } = req.params;
+  const { address } = req.params;
+  client.messages
+    .create({
+      body: `${address}`,
+      to: `+1${phoneNumber}`, // Text this number
+      from: '+18455529187' // From a valid Twilio number
```

Comment:
This should be an environment variable so that any developer can use their own twilio account.